### PR TITLE
Disable multibyte in Sabberworm

### PR DESCRIPTION
**Pull request by bezumkin**
Possible fix for speed issues described in https://github.com/meenie/munee/issues/78

### CODE REVIEW

File: src/Munee/Asset/Type/Css.php [hunk 1]
```diff
@@ -168,7 +169,8 @@ protected function isScss($file)
      */
     protected function fixRelativePaths($content, $originalFile)
     {
-        $cssParser = new CssParser($content);
+        $oSettings = CssSettings::create()->withMultibyteSupport(false);
```
Comment (meenie) [inline src/Munee/Asset/Type/Css.php]: @bezumkin: Could you please rename this variable to something like `$cssParserSettings`. I don't use `o`, `s`, `i` prefixes :).  I'm going to merge this, do a patch release, and then fix it better this weekend.

Comment (bezumkin) [inline src/Munee/Asset/Type/Css.php]: I did`t knew that.

This strings was taken directly from [Sabberworm readme](https://github.com/sabberworm/PHP-CSS-Parser). Next commit renames them as you want.

**Comment by meenie** [comment]
@bezumkin: That's fine :).  Can you squash your commits and force push? So the commit history is clean :).  If you can't, that's fine.

**Comment by bezumkin** [comment]
I got a new commit https://github.com/meenie/munee/pull/83

**Pull request closed without merging**